cbor/examples/cbor/README.md

```diff
@@ -0,0 +1,59 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C6 | ESP32-H2 | ESP32-S2 | ESP32-S3 |
```

Comment:
This is pretty high level example and it should be supported for all targets. Shall we remove this line?

Reply:
Removed